src/ResourceManager/Network/Commands.Network/ServiceEndpointPolicy/GetAzureServiceEndpointPolicyCommand.cs

```diff
@@ -0,0 +1,51 @@
+﻿// ----------------------------------------------------------------------------------
+//
+// Copyright Microsoft Corporation
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+namespace Microsoft.Azure.Commands.Network
+{
+    using Microsoft.Azure.Commands.Network.Models;
+    using Microsoft.Azure.Commands.ResourceManager.Common.ArgumentCompleters;
+    using System.Management.Automation;
+
+    [Cmdlet(VerbsCommon.Get, "AzureRmServiceEndpointPolicy"), OutputType(typeof(PSServiceEndpointPolicy))]
+    public class GetAzureServiceEndpointPolicyCommand : ServiceEndpointPolicyBaseCmdlet
```

Comment:
Is there no list feature when getting an endpoint policy? Like if the user only specified
* RG only : the user gets all the policiess in the RG
* No RG and No name : All the policies in the subscription.